float32 angle
float32 radians
---
int8 status